src/worker.ts

```diff
@@ -1,11 +1,13 @@
 import "unfetch/polyfill";
-import { CONFIG_URL } from "./constants";
+import { CONFIG_PATH } from "./constants";
 import chooseRandomTasks from "./lib/chooseRandomTasks";
 import retry from "./util/promiseRetry";
 import sequence from "./util/promiseSequence";
 import { create as createTask } from "./tasks";
 import Task from "./tasks/task";
 
+declare const PRODUCTION: string;
```

Comment:
Not that familiar with `declare const` can you enlighten me what's going on here?

Reply:
Good question. We are using the WebPack [`DefinePlugin`](https://webpack.js.org/plugins/define-plugin/) to define global variable values at compile time. Therefore, we need to also tell typescript that the global const `PRODUCTION` exists or else it will error. However, I have just noticed this should be a `boolean` and not a `string`. I will also add a comment to avoid confusion.